File: freight-triage-polyglot-ledger-recovery/environment/reconcile/internal/tablex/zone_05.go
package tablex

// zoneFill05 appends zone rows 300..319.
func zoneFill05(out []ZoneRow) []ZoneRow {
	out = append(out, ZoneRow{ZoneKey: "FZ-300", Abbrev: "Z00O", OffsetMinutes: -480, DstShiftMinutes: 60, Hub: "MSP"})
	out = append(out, ZoneRow{ZoneKey: "FZ-301", Abbrev: "Z01P", OffsetMinutes: -420, DstShiftMinutes: 0, Hub: "NSH"})
	out = append(out, ZoneRow{ZoneKey: "FZ-302", Abbrev: "Z02Q", OffsetMinutes: -360, DstShiftMinutes: 0, Hub: "OKC"})
	out = append(out, ZoneRow{ZoneKey: "FZ-303", Abbrev: "Z03R", OffsetMinutes: -300, DstShiftMinutes: 0, Hub: "PDX"})
	out = append(out, ZoneRow{ZoneKey: "FZ-304", Abbrev: "Z04S", OffsetMinutes: -240, DstShiftMinutes: 60, Hub: "PHX"})
	out = append(out, ZoneRow{ZoneKey: "FZ-305", Abbrev: "Z05T", OffsetMinutes: -210, DstShiftMinutes: 0, Hub: "RNO"})
	out = append(out, ZoneRow{ZoneKey: "FZ-306", Abbrev: "Z06U", OffsetMinutes: -180, DstShiftMinutes: 0, Hub: "SLC"})
	out = append(out, ZoneRow{ZoneKey: "FZ-307", Abbrev: "Z07V", OffsetMinutes: -120, DstShiftMinutes: 0, Hub: "SEA"})
	out = append(out, ZoneRow{ZoneKey: "FZ-308", Abbrev: "Z08W", OffsetMinutes: -60, DstShiftMinutes: 60, Hub: "STL"})
	out = append(out, ZoneRow{ZoneKey: "FZ-309", Abbrev: "Z09X", OffsetMinutes: 0, DstShiftMinutes: 0, Hub: "TPA"})
	out = append(out, ZoneRow{ZoneKey: "FZ-310", Abbrev: "Z10Y", OffsetMinutes: 60, DstShiftMinutes: 0, Hub: "YYZ"})
	out = append(out, ZoneRow{ZoneKey: "FZ-311", Abbrev: "Z11Z", OffsetMinutes: 120, DstShiftMinutes: 0, Hub: "YVR"})
	out = append(out, ZoneRow{ZoneKey: "FZ-312", Abbrev: "Z12A", OffsetMinutes: 180, DstShiftMinutes: 60, Hub: "ATL"})
	out = append(out, ZoneRow{ZoneKey: "FZ-313", Abbrev: "Z13B", OffsetMinutes: 210, DstShiftMinutes: 0, Hub: "BOS"})
	out = append(out, ZoneRow{ZoneKey: "FZ-314", Abbrev: "Z14C", OffsetMinutes: 240, DstShiftMinutes: 0, Hub: "CHI"})
	out = append(out, ZoneRow{ZoneKey: "FZ-315", Abbrev: "Z15D", OffsetMinutes: 270, DstShiftMinutes: 0, Hub: "DFW"})
	out = append(out, ZoneRow{ZoneKey: "FZ-316", Abbrev: "Z16E", OffsetMinutes: 300, DstShiftMinutes: 60, Hub: "DEN"})
	out = append(out, ZoneRow{ZoneKey: "FZ-317", Abbrev: "Z17F", OffsetMinutes: 330, DstShiftMinutes: 0, Hub: "DTW"})
	out = append(out, ZoneRow{ZoneKey: "FZ-318", Abbrev: "Z18G", OffsetMinutes: 345, DstShiftMinutes: 0, Hub: "HOU"})
	out = append(out, ZoneRow{ZoneKey: "FZ-319", Abbrev: "Z19H", OffsetMinutes: 360, DstShiftMinutes: 0, Hub: "IND"})
	return out
}

File: freight-triage-polyglot-ledger-recovery/environment/native/src/tables/zone_05.cpp
#include "freight/tables.h"

namespace freight {

// zone table rows 300..319.
void zoneTableFill05(std::vector<ZoneTableRow>* out) {
  out->push_back(ZoneTableRow{"FZ-300", "Z00O", -480, 60, "MSP"});
  out->push_back(ZoneTableRow{"FZ-301", "Z01P", -420, 0, "NSH"});
  out->push_back(ZoneTableRow{"FZ-302", "Z02Q", -360, 0, "OKC"});
  out->push_back(ZoneTableRow{"FZ-303", "Z03R", -300, 0, "PDX"});
  out->push_back(ZoneTableRow{"FZ-304", "Z04S", -240, 60, "PHX"});
  out->push_back(ZoneTableRow{"FZ-305", "Z05T", -210, 0, "RNO"});
  out->push_back(ZoneTableRow{"FZ-306", "Z06U", -180, 0, "SLC"});
  out->push_back(ZoneTableRow{"FZ-307", "Z07V", -120, 0, "SEA"});
  out->push_back(ZoneTableRow{"FZ-308", "Z08W", -60, 60, "STL"});
  out->push_back(ZoneTableRow{"FZ-309", "Z09X", 0, 0, "TPA"});
  out->push_back(ZoneTableRow{"FZ-310", "Z10Y", 60, 0, "YYZ"});
  out->push_back(ZoneTableRow{"FZ-311", "Z11Z", 120, 0, "YVR"});
  out->push_back(ZoneTableRow{"FZ-312", "Z12A", 180, 60, "ATL"});
  out->push_back(ZoneTableRow{"FZ-313", "Z13B", 210, 0, "BOS"});
  out->push_back(ZoneTableRow{"FZ-314", "Z14C", 240, 0, "CHI"});
  out->push_back(ZoneTableRow{"FZ-315", "Z15D", 270, 0, "DFW"});
  out->push_back(ZoneTableRow{"FZ-316", "Z16E", 300, 60, "DEN"});
  out->push_back(ZoneTableRow{"FZ-317", "Z17F", 330, 0, "DTW"});
  out->push_back(ZoneTableRow{"FZ-318", "Z18G", 345, 0, "HOU"});
  out->push_back(ZoneTableRow{"FZ-319", "Z19H", 360, 0, "IND"});
}

}  // namespace freight
